Source/Serilog.Exceptions.SqlServer/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
```

Comment:
Apparently there is a way to use `InternalsVisibleTo` in csproj. Someone commented in my blog post 'Cleaning up csproj'.